### ISSUE TITLE
Feats > Two Weapon Defense > Feat YML

### ISSUE THREAD

**Issue by ucffool** [openlegend/core-rules#331]
For YML standardization, the following should be changed:
Attack Specialization ~~(Any one-handed weapon)~~

It should only refer to the exact FEAT by name; any additional notes regarding it being one-handed should be part of the description. Otherwise it becomes the exception against the standardized layout.

The reason this is obvious to me is that when filtering by prerequisites, one would need to select "Attack Specialization (Any one-handed weapon)" which doesn't exist or if it was done automatically (say in a character builder), you would need to look at the list of known Feats and then filter by them, except it wouldn't work due to the extra text.

In other words, it 100% breaks the relational aspects of the feats.yml.

**Comment by ucffool** [comment]
Fixed and closed #275bc6e
